index.html

```diff
@@ -78,13 +77,65 @@
     </main>
     <footer class="footer">
         <div class="wrapper footer__wrapper">
+            <span class="icon icon_settings"></span>
+            <div class="settings">
+                <div class="settings__header">
+                    <div class="change change__language">
+                        <p class="title title__language">Change language</p>
+                        <select class="language">
+                            <option value="en">english</option>
+                            <option value="ru">russian</option>
+                        </select>
+                    </div>
+                    <div class="change settings__photo">
+                        <p class="title title__photo">Change Photo Source</p>
+                        <select class="photo">
+                            <option value="github">GitHub</option>
+                            <option value="unsplash">Unsplash Api</option>
+                            <option value="flickr">Flickr Api</option>
+                        </select>
+                    </div>
+                </div>
+                <div class="settings__blocks">
+                    <div class="settings__blocks__wrapper">
```

Comment:
this also goes against BEM. you can't have double element like `block__element__element`. you can just name it like
```
settings__blocks-wrapper
```